components/LinkButton.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC } from 'react'
+
+export interface LinkButton {
+  href?: string
+  text: string
+}
+
+const LinkButton: FC<LinkButton> = ({ href, text }) => {
+  return (
+    <a
```

Comment:
The example you linked here is exactly what I'm suggesting (have the LinkButton component be almost identical to the example linked)
https://github.com/DTS-STN/passport-status/pull/45#issuecomment-1268612688

Reply:
In that case it would be 
`<outerComponent>
<Link passHref>
<componentWrappingAtag><a></a></componentWrappingAtag>
</Link>
</outerComponent>`
?

Originally I had it like that but I thought it too convoluted